app/routes/teams.rb

```diff
@@ -2,6 +2,18 @@ module ExercismWeb
   module Routes
     # rubocop:disable Metrics/ClassLength
     class Teams < Core
+      get '/teams/search' do
```

Comment:
This is going to conflict with [this](https://github.com/exercism/exercism.io/blob/master/app/routes/teams.rb#L156) route, should we change this to something like `teams/new` or rename `/teams/search` to something like `teams/list`?